DevoxxClientMobile/src/main/java/com/devoxx/views/helper/Util.java

```diff
@@ -199,4 +207,38 @@ public static void showToast(String message, Duration duration)  {
     public static String safeStr(String s) {
         return s == null? "": s.trim();
     }
+
+    public static void hidePastConferenceMessage() {
+        if (pastConferenceMessage != null) {
+            pastConferenceMessage.hide();
+        }
+    }
+
+    public static void showPastConferenceMessage() {
+        if (pastConferenceMessage == null) {
```

Comment:
Another possible way to show "past conference" could be the navigation drawer The current header has a big image and we could add some text to the right corner, rotated 45º, saying "ENDED" or alike over a dark background. Just an idea.

Reply:
It is a good idea. We don't even have to rotate it by 45 degree and can just show it on the top of the header. The only thing that worries me is that it won't be very prominent to the user defeating its purpose.

Reply:
I was thinking of a corner ribbon, something like [this](https://camo.githubusercontent.com/0ee888cc5b6f2fa372ded87a6a01eb2789ba88dd/68747470733a2f2f7261772e6769746875622e636f6d2f64696e673238302f646576656c6f706d656e745f726962626f6e2f6d61737465722f6578616d706c655f696d6167652e706e67).

Reply:
I just logged #124

Reply:
ok